install/sample/base/greeting.yaml

```diff
@@ -33,54 +33,3 @@ data:
           }
       }
     }
----
-apiVersion: apps/v1beta1
```

Comment:
Just make sure the demosinglecluster overlay works as expected with this change.